alleycats-core/src/main/scala/alleycats/std/iterable.scala

```diff
@@ -0,0 +1,38 @@
+package alleycats
+package std
+
+import cats.{Eval, Foldable}
+import export._
+
+@reexports(IterableInstances)
+object iterable extends LegacyIterableInstances
+
+@exports
+object IterableInstances {
+  @export(Orphan)
+  implicit val exportIterableFoldable: Foldable[Iterable] =
+    new Foldable[Iterable] {
+      override def foldLeft[A, B](fa: Iterable[A], b: B)(f: (B, A) => B): B = fa.foldLeft(b)(f)
+
+      // based upon foldRight of List in Cats
+      override def foldRight[A, B](fa: Iterable[A], lb: Eval[B])(f: (A, Eval[B]) => Eval[B]): Eval[B] = {
+        def loop(as: Iterable[A]): Eval[B] =
```

Comment:
I think this can actually be faster and get code-reuse if we use `Foldable.iterateRight`: https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/Foldable.scala#L508